rules/opentitan/ci.bzl

```diff
@@ -0,0 +1,47 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@bazel_skylib//lib:sets.bzl", "sets")
+
+# List of execution environments that are always enabled in CI.
+_ALWAYS_RUN_IN_CI = [
+    "//hw/top_earlgrey:sim_verilator",
+    "//hw/top_earlgrey:sim_dv",
+]
+
+# List of execution environments among which only one should run. The list
+# is sorted by priority: the first available will be picked.
+_ONLY_RUN_ONE_IN_CI_SORTED = [
+    "//hw/top_earlgrey:fpga_cw310_sival_rom_ext",
+    "//hw/top_earlgrey:fpga_cw310_rom_ext",
+    "//hw/top_earlgrey:fpga_cw310_sival",
+    "//hw/top_earlgrey:fpga_cw310_rom_with_fake_keys",
+    "//hw/top_earlgrey:fpga_cw310_test_rom",
+    "//hw/top_earlgrey:fpga_cw340_sival_rom_ext",
+    "//hw/top_earlgrey:fpga_cw340_sival",
+    "//hw/top_earlgrey:fpga_cw340_rom_ext",
+    "//hw/top_earlgrey:fpga_cw340_rom_with_fake_keys",
+    "//hw/top_earlgrey:fpga_cw340_test_rom",
```

Comment:
I think we want to put the CW340s on top of the CW310s here

Reply:
Do we have enough CW340 runners? Because this will switch almost the entire loads to the CW340

Reply:
I’ve added two more so we have 6 CW340s which should be enough.

Reply:
I'll swap the order of the CW310s and CW340s later when I enable more tests.